internal/hud/hud.go

```diff
@@ -83,42 +86,32 @@ func (h *Hud) Run(ctx context.Context, st *store.Store, refreshRate time.Duratio
 			} else {
 				return nil
 			}
-		case ready := <-a.readyCh:
-			screenEvents, err = h.r.SetUp(ready, a.winchCh)
-			if err != nil {
-				return err
-			}
-			h.Refresh(ctx)
-		case <-a.winchCh:
+		case <-sigwinchCh:
```

Comment:
pretty sure that if you're using the regular `tcell.NewScreen` (rather than the hacky `NewScreenFromTty`, which we built) it takes care of refreshing on sigwinch for you so you don't need this channel/case anymore (see `tcell.defaultSigwinch` and any call to `t.sigwinch.Notify`).

Reply:
Ah, I figured tcell could only take care of refreshing tcell itself on sigwinch, since it doesn't have a hook to call back into rty code so that rty can re-render with the new dimensions. It turns out tcell emits resizes as a `tcell.Event`. Thanks!